react-app/src/library/services/headset.ts

```diff
@@ -230,17 +249,13 @@ export default class HeadsetService {
         'An error has occurred while trying to establish a connection',
         this.selectedImplementation?.errorCode
       );
-      return `Error occurred while establishing connection`;
-      // return `dummy.connectionStatus.error`
+      return `dummy.connectionStatus.error`
```

Comment:
I don't remember what I was looking at before but I believe I was looking at another repo to help build this out which might have used the same terminology.  I will change that though because it's definitely not proper